codemirror-wp.php

```diff
@@ -250,6 +275,9 @@ public static function customize_controls_print_footer_scripts() {
 		#customize-control-custom_css .CodeMirror {
 			height: calc( 100vh - 185px );
 		}
+		.CodeMirror-lint-tooltip {
+			z-index: 500000;
```

Comment:
The reason for this is https://github.com/WordPress/wordpress-develop/blob/0b5b1a667f28c76cbc9db3d2a16ced99353f6f77/src/wp-admin/css/themes.css#L1323